src/foam/u2/crunch/WizardRunner.js

```diff
@@ -26,25 +26,43 @@ foam.CLASS({
       documentation: `
         Either a capability, string, or WizardFlow
       `
-    }
+    },
+    {
+      name:'options',
+      class: 'Map',
+    },
+    {
+      name:'sequence',
+      factory: function() {
+        const IN_PROGRESS = this.crunchController.WizardStatus.IN_PROGRESS;
+        const seq = this.getSequence_(this.__context__, this.isInline);
+        return seq
+      } 
+    },
+    {
+      name: 'parentWizard',
+      factory: function() {
+        return this.getParentWizard_();
+      }
+    },
+    {
+      name: 'isInLine',
+      expression: function(parentWizard, options) {
+        return (options.inline ?? true) && !! parentWizard;
+      }
+    },
   ],
   methods: [
-    async function launch(x, options) {
-      options = options || {};
```

Comment:
Would it be better to keep this and default it to `this.options` instead, that way you dont need a new wizardRunner everytime you need different options?

Reply:
We need to be able to make configurations on the sequence before calling `launch`, and the sequence will depend on the options. So no, in this case we're making the decision that an instance of WizardRunner is - semantically - a configuration you want to use (and potentially modify) to run a wizard.

Reply:
Okay, can we add this to the documentation for this prop so that i dont do this exact thing in 4 months?

Reply:
@sarthak-marwaha this has been added